src/RangeToFilter.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Modules\Core\Filters;
```

Comment:
Thank you for your kind words and for the brilliant idea behind this package!

You’re absolutely right about the namespace; consistency is crucial to avoid confusion or breaking changes. Here are my updates based on your feedback to ensure we maintain uniformity across the filters. Let me know your thoughts or if there’s anything else you’d like adjusted!

@l3aro